enjarify.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Do we really want to include this file in versioning ?
I would rather have a dedicated tools subdirectory, and store all of them as a zip file